docs/blueprints.rst

```diff
@@ -176,16 +176,24 @@ the `template_folder` parameter to the :class:`Blueprint` constructor::
 
     admin = Blueprint('admin', __name__, template_folder='templates')
 
-As for static files, the path can be absolute or relative to the blueprint
-resource folder.  The template folder is added to the searchpath of
-templates but with a lower priority than the actual application's template
-folder.  That way you can easily override templates that a blueprint
-provides in the actual application.
+For static files, the path can be absolute or relative to the blueprint
+resource folder.  
+
+The template folder is added to the search path of templates but with a lower 
+priority than the actual application's template folder. That way you can 
```

Comment:
What about adding an additional sentence here explaining how the priority order is resolved when two blueprint templates conflict?